src/blocks/share/index.php

```diff
@@ -306,9 +308,13 @@ function coblocks_register_share_block() {
 					'type'    => 'boolean',
 					'default' => false,
 				),
+				'socialProfileAttributes'		=> array(
```

Comment:
I would avoid adding a new attribute to a block for the purpose of different block. I understand this is for transforming back to the social-profile block. The method works and I don't think we absolutely have to change this.

My initial thought is to use localStorage to persist the urls during the transform so we can read them back. We can read them back if we stay in the editor. I think it's a valid expectation that if you leave the editor you've "finalized" the edit and wouldn't expect the urls to be remembered once transformed into the social block.

Reply:
Rewrote transforms and removed cross-block attributes in favor of localStorage use.